Core/src/org/sleuthkit/autopsy/coreutils/ExecUtil.java

```diff
@@ -213,6 +214,40 @@ public static void killProcess(Process process) {
     }
 
     /**
+     * Shuts down a task executor service, waiting until all tasks are
+     * terminated. The current policy is to wait for the tasks to finish so that
+     * the case for which the executor is running can be left in a consistent
+     * state.
+     *
+     * @param executor The executor.
+     */
+    public static void shutDownTaskExecutor(ExecutorService executor) {
```

Comment:
@wschaeferB and @APriestman, good job factoring this code out of the Case class for general use! However, ExecUtil is documented as "Executes a command line using an operating system process with a configurable timeout and pluggable logic to kill or continue the process on timeout."  We should make a new utility class for this code, ThreadUtils or something. We could perhaps even add another method right away for the intuitive ThreadFactoryBuilder stuff we do all over the place, what do you think?